modules/core/src/main/java/com/illposed/osc/LibraryInfo.java

```diff
@@ -286,6 +288,19 @@ public static boolean isAndroidApi23AndLower() {
 		}
 	}
 
+	/**
+	 * Checks if java.awt.color is available and the library is used on desktops
+	 */
+	public static boolean isDesktop() {
+		try {
+			Class.forName("java.awt.color");
```

Comment:
should it not be `Color` instead of `color`?

Reply:
You are probably right ;-)